app/models/service_template_ansible_playbook.rb

```diff
@@ -151,4 +151,17 @@ def update_catalog_item(options, auth_user = nil)
     end
     super
   end
+
+  def destroy(auth_user = 'system')
+    config_info = options[:config_info]
+    [:provision, :retirement, :reconfigure].each do |action|
```

Comment:
This is not the right way to clean job templates. We should iterate resource_actions and get the job template from `resource_action.configuration_template`